Common/Common-DateTime.yaml

```diff
@@ -0,0 +1,51 @@
+HourRegex: !simpleRegex
```

Comment:
Nit: I'd prefer to call this directory Resources/Patterns or something like it. Common implies common across languages and this is actually common resources to be compiled/transpiled into code in different programming languages.